### ISSUE TITLE
commonj.sdo:commonj.sdo:jar:2.1.1.v201112051852 is missing 

### ISSUE THREAD

**Issue by sco0ter** [chenjianjx/wsdl2html#34]
When building with Maven, I get the following error:

```
[WARNING] The POM for commonj.sdo:commonj.sdo:jar:2.1.1.v201112051852 is missing, no dependency information available
Downloading from central: https://repo.maven.apache.org/maven2/commonj/sdo/commonj.sdo/2.1.1.v201112051852/commonj.sdo-2.1.1.v201112051852.jar

[ERROR] Failed to execute goal on project ***: Could not resolve dependencies for project ***T: Failure to find commonj.sdo:commonj.sdo:jar:2.1.1.v201112051852 in https://repo.maven.apache.org/maven2/ was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
```

Indeed I cannot find the artifact on Maven Central (although I can find it on https://mvnrepository.com).

I have solved it by exclusion, without further issues:

```
<dependency>
            <groupId>com.github.chenjianjx</groupId>
            <artifactId>wsdl2html</artifactId>
            <version>4.2.0</version>
            <exclusions>
                <exclusion>
                    <groupId>commonj.sdo</groupId>
                    <artifactId>commonj.sdo</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
```

Can you please elaborate why this dependency is needed and either remove it or change it it:

```
<dependency>
  <groupId>org.eclipse.persistence</groupId>
  <artifactId>commonj.sdo</artifactId>
  <version>2.1.1</version>
</dependency>
```
This feels like the more official version.

**Comment by chenjianjx** [comment]
Mate, before you create an issue,  you need to check whether this is a problem of your own project.

**Comment by sco0ter** [comment]
Here's the dependency tree. The dependency in question is pulled from `com.sun.xml.ws:jaxws-ri:pom:2.3.0` from your artifact.
It's certainly not your fault, not sure, why it is not found on Maven Central.

```
com.github.chenjianjx:wsdl2html:jar:4.2.0:compile
   commons-lang:commons-lang:jar:2.6:compile
   commons-io:commons-io:jar:2.4:compile
   org.freemarker:freemarker:jar:2.3.20:compile
   com.sun.xml.ws:jaxws-ri:pom:2.3.0:compile
      javax.annotation:javax.annotation-api:jar:1.3:compile
      com.sun.org.apache.xml.internal:resolver:jar:20050927:compile
      com.sun.xml.ws:jaxws-rt:jar:2.3.0:compile
         com.sun.xml.bind:jaxb-core:jar:2.3.0:compile
         com.sun.xml.bind:jaxb-impl:jar:2.3.0:compile
         org.glassfish.gmbal:gmbal-api-only:jar:3.1.0-b001:compile
         org.codehaus.woodstox:woodstox-core-asl:jar:4.4.1:runtime
      com.sun.xml.ws:jaxws-tools:jar:2.3.0:compile
         com.sun.xml.bind:jaxb-xjc:jar:2.3.0:compile
         com.sun.xml.bind:jaxb-jxc:jar:2.3.0:compile
      com.sun.xml.ws:jaxws-eclipselink-plugin:jar:2.3.0:compile
         org.eclipse.persistence:eclipselink:jar:2.6.4:compile
            org.eclipse.persistence:javax.persistence:jar:2.1.1:compile
            org.eclipse.persistence:commonj.sdo:jar:2.1.1:compile
            javax.validation:validation-api:jar:2.0.1.Final:compile (sc
            org.glassfish:javax.json:jar:1.0.4:compile
      com.sun.xml.ws:sdo-eclipselink-plugin:jar:2.3.0:compile
         commonj.sdo:commonj.sdo:jar:2.1.1.v201112051852:compile
      com.sun.xml.ws:release-documentation:zip:docbook:2.3.0:compile
      com.sun.xml.ws:samples:zip:2.3.0:compile
      javax.xml.ws:jaxws-api:jar:2.3.0:compile
      javax.xml.bind:jaxb-api:jar:2.3.0:compile
      javax.xml.soap:javax.xml.soap-api:jar:1.4.0:compile
      javax.jws:jsr181-api:jar:1.0-MR1:compile
   javax.activation:javax.activation-api:jar:1.2.0:compile

```

But appearently it has been addressed:
Simply upgrading to 2.3.3 should fix the issue.

Compare
https://search.maven.org/artifact/com.sun.xml.ws/sdo-eclipselink-plugin/2.3.3/jar
to
https://search.maven.org/artifact/com.sun.xml.ws/sdo-eclipselink-plugin/2.3.0/jar

(it references different commonj.sdo groupIds)

**Comment by sco0ter** [comment]
I also believe you don't need all that stuff from `jaxws-ri`, like `samples`, `release-documentation` etc. It only blows up the application, where your library is used it. I only use `jaxws-tools` now and exclude the whole `jaxws-ri` and it still works like a charm.
Please consider it.

```
<dependency>
            <groupId>com.github.chenjianjx</groupId>
            <artifactId>wsdl2html</artifactId>
            <version>4.2.0</version>
            <exclusions>
                <exclusion>
                    <groupId>com.sun.xml.ws</groupId>
                    <artifactId>jaxws-ri</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
        <dependency>
            <groupId>com.sun.xml.ws</groupId>
            <artifactId>jaxws-tools</artifactId>
            <version>2.3.0</version>
        </dependency>
```

**Comment by chenjianjx** [comment]
Yes.  You are right.   It's been fixed in  v 4.2.1.  Only using jaxws-tools now
